nextcord/flags.py

```diff
@@ -486,6 +487,58 @@ def all(self) -> List[UserFlags]:
         return [public_flag for public_flag in UserFlags if self._has_flag(public_flag.value)]
 
 
+@fill_with_flags()
+class MemberFlags(BaseFlags):
+    r"""Wraps up the Discord Guild Member flags
+
+    .. versionadded:: 2.5
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two MemberFlags are equal.
+
+        .. describe:: x != y
+
+            Checks if two MemberFlags are not equal.
+        .. describe:: hash(x)
+
+            Return the flag's hash.
+        .. describe:: iter(x)
+
+            Returns an iterator of ``(name, value)`` pairs. This allows it
+            to be, for example, constructed as a dict or a list of pairs.
+            Note that aliases are not shown.
+
+    Attributes
+    -----------
+    value: :class:`int`
+        The raw value. You should query flags via the properties
+        rather than using this raw value.
+    """
+
+    @flag_value
+    def did_rejoin(self):
+        """:class:`bool`: Returns ``True`` if the member left and rejoined the :attr:`~nextcord.Member.guild`."""
```

Comment:
Why are you referencing the member's guild attribute and not the guild class?

Reply:
Because that's the relevant guild I think.

Reply:
It doesn't make sense to reference the member's guild attribute here. Just refer to the guild class instead.

Reply:
Done!